src/civic/index.js

```diff
@@ -649,64 +667,67 @@ const upload = async ({
         let mappedCount = 0;
         const postupload = [];
 
-        // resolve combinations
+        // Nested loop on recordList. Resolve combinations
         for (const record of recordList) {
-            record.variants = [varById[record.variant_id]]; // OR-ing of variants
-
-            if (record.drugs === undefined || record.drugs.length === 0) {
+            if (record.drugs === null || record.drugs.length === 0) {
                 record.drugs = [null];
-            } else if (record.drug_interaction_type === 'Combination' || record.drug_interaction_type === 'Sequential') {
+            } else if (
+                record.drugInteractionType === 'COMBINATION'
+                || record.drugInteractionType === 'SEQUENTIAL'
+            ) {
                 record.drugs = [record.drugs];
-            } else if (record.drug_interaction_type === 'Substitutes' || record.drugs.length < 2) {
+            } else if (record.drugInteractionType === 'SUBSTITUTES' || record.drugs.length < 2) {
                 record.drugs = record.drugs.map(drug => [drug]);
-                record.drug_interaction_type = null;
+                record.drugInteractionType = null;
             } else {
-                logger.error(`(evidence: ${record.id}) unsupported drug interaction type (${record.drug_interaction_type}) for a multiple drug (${record.drugs.length}) statement`);
+                logger.error(`(evidence: ${record.id}) unsupported drug interaction type (${record.drugInteractionType}) for a multiple drug (${record.drugs.length}) statement`);
                 counts.skip++;
                 continue;
             }
 
             let orCombination;
 
-            if (orCombination = /^([a-z]\d+)([a-z])\/([a-z])$/i.exec(record.variants[0].name)) {
+            if (orCombination = /^([a-z]\d+)([a-z])\/([a-z])$/i.exec(record.variant.name)) {
                 const [, prefix, tail1, tail2] = orCombination;
-                record.variants = [
-                    { ...record.variants[0], name: `${prefix}${tail1}` },
-                    { ...record.variants[0], name: `${prefix}${tail2}` },
+                record.variant = [
```

Comment:
This is incorrect, variants and variant are separate entities. We need the nested loop in the next part. this is important when for example we have a civic evidence item with a variant: "K12G or K12M". In graphkb this must be split into 2 different statements since we do not allow OR combinations. This is why it is stored on "variants" and not "variant" and looped over in the code below

Reply:
ok. Rolled back to previous implementation. I thought it was related to an older implementation of variant querrying.